ports/stm32/boards/VCC_GND_F407VE/mpconfigboard.h

```diff
@@ -0,0 +1,152 @@
+#define MICROPY_HW_BOARD_NAME       "VCC-GND STM32F407VE"
+#define MICROPY_HW_MCU_NAME         "STM32F407VE"
+#define MICROPY_HW_FLASH_FS_LABEL   "VCCGNDF407VE"
+
+// 1 = use internal flash (512 KByte)
+// 0 = use external SPI flash
+#define MICROPY_HW_ENABLE_INTERNAL_FLASH_STORAGE (1)
+
+#define MICROPY_HW_HAS_FLASH        (1)
+#define MICROPY_HW_ENABLE_RNG       (1)
+#define MICROPY_HW_ENABLE_RTC       (1)
+#define MICROPY_HW_ENABLE_DAC       (1)
+#define MICROPY_HW_ENABLE_USB       (1)
+#define MICROPY_HW_ENABLE_SDCARD    (1)
+
+// HSE is 25MHz
+#define MICROPY_HW_CLK_PLLM (25) // divide external clock by this to get 1MHz
+#define MICROPY_HW_CLK_PLLN (336) // PLL clock in MHz
+#define MICROPY_HW_CLK_PLLP (RCC_PLLP_DIV2) // divide PLL clock by this to get core clock
+#define MICROPY_HW_CLK_PLLQ (7) // divide core clock by this to get 48MHz
+
+// The board has a 32kHz crystal for the RTC
+#define MICROPY_HW_RTC_USE_LSE      (1)
+#define MICROPY_HW_RTC_USE_US       (0)
+// #define MICROPY_HW_RTC_USE_CALOUT   (1)  // turn on/off PC13 512Hz output
+
+// USART1
+#define MICROPY_HW_UART1_TX     (pin_A9)   // PA9,PB6
+#define MICROPY_HW_UART1_RX     (pin_A10)  // PA10,PB7
+
+// USART2
+#define MICROPY_HW_UART2_TX     (pin_A2)  // PA2,PD5
+#define MICROPY_HW_UART2_RX     (pin_A3)  // PA3,PD6
+#define MICROPY_HW_UART2_RTS    (pin_A1)  // PA1,PD4
+#define MICROPY_HW_UART2_CTS    (pin_A0)  // PA0,PD3
+
+// USART3
+#define MICROPY_HW_UART3_TX     (pin_D8)  // PB10,PC10,PD8
+#define MICROPY_HW_UART3_RX     (pin_D9)  // PB11,PC11,PD9
+#define MICROPY_HW_UART3_RTS    (pin_D12) // PB14,PD12
+#define MICROPY_HW_UART3_CTS    (pin_D11) // PB13,PD11
+
+// UART4
+#define MICROPY_HW_UART4_TX     (pin_A0)  // PA0,PC10
+#define MICROPY_HW_UART4_RX     (pin_A1)  // PA1,PC11
+
+// UART5
+#define MICROPY_HW_UART5_TX     (pin_C12) // PC12
+#define MICROPY_HW_UART5_RX     (pin_D2)  // PD2
+
+// USART6
+#define MICROPY_HW_UART6_TX     (pin_C6) // PC6,PG14
+#define MICROPY_HW_UART6_RX     (pin_C7) // PC7,PG9
+
+// I2C busses
+#define MICROPY_HW_I2C1_SCL (pin_B6)  // PB8,PB6
+#define MICROPY_HW_I2C1_SDA (pin_B7)  // PB9,PB7
+#define MICROPY_HW_I2C2_SCL (pin_B10) // PB10
+#define MICROPY_HW_I2C2_SDA (pin_B11) // PB11
+#define MICROPY_HW_I2C3_SCL (pin_A8)  // PA8
+#define MICROPY_HW_I2C3_SDA (pin_C9)  // PC9
```

Comment:
IMO it'd be a bit neater to align the `(pin_xxx)` parts to the same column as for UART

Reply:
Done